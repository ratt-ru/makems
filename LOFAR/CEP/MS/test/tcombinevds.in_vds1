Name       = /Users/diepen/LOFAR.MS1
FileSys    = /Users/diepen
ClusterDesc= /Users/diepen/cdesc.data
StartTime  = 2007/07/27/16:04:49
EndTime    = 2007/07/28/16:04:48
StepTime   = 30.199
NChan      = [8,10]
StartFreqs = [10,20]
EndFreqs   = [20,30]
Extra.CorrNames=[XX,XY,YX,YY]
Extra.DataCubeShape=[4,1,389096]
Extra.DataFileIsRegular=1
Extra.DataFileName=/Users/diepen/LOFAR.MS/table.f2_TSM1
Extra.DataTileShape=[4,1,4096]
Extra.Stationnames=[CS010_dipole0,CS010_dipole4,CS010_dipole8,CS010_dipole12,CS001_dipole0,CS001_dipole4,CS001_dipole8,CS001_dipole12,CS008_dipole0,CS008_dipole4,CS008_dipole8,CS008_dipole12,CS016_dipole0,CS016_dipole4,CS016_dipole8,CS016_dipole12]
