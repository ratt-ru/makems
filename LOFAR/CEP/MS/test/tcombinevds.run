#!/bin/sh

../src/combinevds tcombinevds_tmp.vdsout tcombinevds.in_vds1 tcombinevds.in_vds2
# Keywords Name have absolute paths in their values, so remove them if needed.
if test "$1" = 1; then
  cat tcombinevds_tmp.vdsout
else
  grep -v 'Part[0-9]*\..*Name *=' tcombinevds_tmp.vdsout
fi
