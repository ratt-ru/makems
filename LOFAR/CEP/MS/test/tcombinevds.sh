#!/bin/sh
./runctest.sh tcombinevds > tcombinevds.log 2>&1
