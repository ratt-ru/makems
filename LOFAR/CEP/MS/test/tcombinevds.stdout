Name       = tcombinevds_tmp.vdsout
ClusterDesc= /Users/diepen/cdesc.data
StartTime  = 2007/07/27/16:04:49.000
EndTime    = 2007/07/28/16:04:48.000
StepTime   = 30.199
NChan      = [8,10,1,4]
StartFreqs = [10,20,100,110]
EndFreqs   = [20,30,110,150]
NParts = 2
>>>Part0.Name       = tcombinevds.in_vds1
<<<
Part0.FileSys    = /Users/diepen
Part0.ClusterDesc= /Users/diepen/cdesc.data
Part0.StartTime  = 2007/07/27/16:04:49.000
Part0.EndTime    = 2007/07/28/16:04:48.000
Part0.StepTime   = 30.199
Part0.NChan      = [8,10]
Part0.StartFreqs = [10,20]
Part0.EndFreqs   = [20,30]
>>>Part1.Name       = tcombinevds.in_vds2
<<<
Part1.FileSys    = /Users/diepen
Part1.ClusterDesc= /Users/diepen/cdesc.data
Part1.StartTime  = 2007/07/27/16:04:49.000
Part1.EndTime    = 2007/07/28/16:04:48.000
Part1.StepTime   = 30.199
Part1.NChan      = [1,4]
Part1.StartFreqs = [100,110,120,130,140]
Part1.EndFreqs   = [110,120,130,140,150]
